ckanext/publicamundi/themes/geodata/plugin.py

```diff
@@ -187,8 +199,43 @@ class GeodataThemePlugin(plugins.SingletonPlugin):
     plugins.implements(plugins.IRoutes, inherit=True)
     plugins.implements(plugins.IPackageController, inherit=True)
 
-    # ITemplateHelpers
-    
+    def init_contact_captcha(self):
+
+        from wheezy.captcha.image import captcha
```

Comment:
this is a test code section i added while trying to implement a captcha in the contact form, but actually never executes. Will remove in next commit!